var/spack/repos/builtin/packages/py-py4j/package.py

```diff
@@ -13,6 +13,17 @@ class PyPy4j(PythonPackage):
     homepage = "https://www.py4j.org/"
     pypi = "py4j/py4j-0.10.4.zip"
 
+    version(
```

Comment:
Can you add a `url_for_version` so we don't need to keep copying the URL each time? Let me know if you need help.